File: UI/ebarbershop_mobile/lib/models/korisnik_uloga.dart
import 'package:ebarbershop_mobile/models/korisnik.dart';
import 'package:ebarbershop_mobile/models/uloga.dart';
import 'package:json_annotation/json_annotation.dart';
part 'korisnik_uloga.g.dart';
@JsonSerializable()
class KorisnikUloga {
  int? korisnikUlogaId;
  int? korisnikId;
  int? ulogaId;
  Uloga? uloga;
  String? datumDodjele;
  Korisnik? korisnik;

  KorisnikUloga({
    this.korisnikUlogaId,
    this.korisnikId,
    this.ulogaId,
    this.uloga,
    this.datumDodjele,
    this.korisnik
  });

  factory KorisnikUloga.fromJson(Map<String, dynamic> json) => _$KorisnikUlogaFromJson(json);
  Map<String, dynamic> toJson() => _$KorisnikUlogaToJson(this);
}





//  Korisnik Korisnik 

File: eBarbershop.Services/Database/KorisnikUloga.cs
﻿using System;
using System.Collections.Generic;

namespace eBarbershop.Services.Database;

public  class KorisnikUloga
{
    public int KorisnikUlogaId { get; set; }

    public int KorisnikId { get; set; }

    public int UlogaId { get; set; }

    public DateTime DatumDodjele { get; set; }

    public virtual Korisnik Korisnik { get; set; } = null!;

    public virtual Uloga Uloga { get; set; } = null!;
}
